gems/pending/MiqVm/MiqVm.rb

```diff
@@ -63,23 +63,30 @@ def openDisks(diskFiles)
         pVolumes = Array.new
         
         $log.debug "openDisks: no disk files supplied." if !diskFiles
-        
+
         #
         # Build a list of the VM's physical volumes.
         #
         diskFiles.each do |dtag, df|
             $log.debug "openDisks: processing disk file (#{dtag}): #{df}"
             dInfo = OpenStruct.new
-            
+
             if @ost.miqVim
                 dInfo.vixDiskInfo = Hash.new
                 dInfo.vixDiskInfo[:fileName]    = @ost.miqVim.datastorePath(df)
                 if @ost.miqVimVm && @ost.miqVim.isVirtualCenter?
-                    @vdlConnection = @ost.miqVimVm.vdlVcConnection if !@vdlConnection
-                    $log.debug "openDisks (VC): using disk file path: #{dInfo.vixDiskInfo[:fileName]}"
+                  thumb_print    = VcenterThumbPrint.new(@ost.miqVimVm.invObj.server)
```

Comment:
We should probably leave this indented by 4 spaces, because that's what the surrounding file seems to be doing.

If we change this part on its own, it'll get mangled when we do a bulk fix of the indentation.

(And yes, that is probably the exact opposite of what rubocop told you to do :smile:)